diff --git a/node_modules/typescript-react-function-component-props-handler/index.js b/node_modules/typescript-react-function-component-props-handler/index.js
index 5c636b9..ca4c9a0 100644
--- a/node_modules/typescript-react-function-component-props-handler/index.js
+++ b/node_modules/typescript-react-function-component-props-handler/index.js
@@ -47,10 +47,6 @@ function checkForProptypes(path, paramTypeName) {
 }
 
 function setParamsTypeDefinitionFromFunctionType(documentation, path) {
-    if (path.parentPath.node.init && path.parentPath.node.init.params.length === 0) {
-        return;
-    }
-
     if (
         path.node.type === 'ArrowFunctionExpression' &&
         (
